### PR TITLE
remove duplicated validation for service-cluster-ip-range

### PR DESCRIPTION
**Pull request by pandaamanda**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
service-cluster-ip-range is already validated in https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/server.go#L608
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by pandaamanda** [comment]
PTAL.  /assign @caesarxuchao @wojtek-t

File: cmd/kube-apiserver/app/options/options.go [hunk 1]
```diff
@@ -71,6 +71,7 @@ type ServerRunOptions struct {
 	// of parsing ServiceClusterIPRange into actual values
```
Comment (caesarxuchao) [inline cmd/kube-apiserver/app/options/options.go:71]: Can you update this comment?

**Review by caesarxuchao — COMMENTED** [review]
LGTM otherwise.

**Comment by aojea** [comment]
LGTM

File: cmd/kube-apiserver/app/options/options.go (hunk 1 above)
Comment (pandaamanda) [reply cmd/kube-apiserver/app/options/options.go:71]: fixed.

File: cmd/kube-apiserver/app/options/options.go [hunk 2]
```diff
@@ -69,8 +69,10 @@ type ServerRunOptions struct {
 	ServiceClusterIPRanges string
 	// PrimaryServiceClusterIPRange and SecondaryServiceClusterIPRange are the results
 	// of parsing ServiceClusterIPRange into actual values
+	// APIServerServiceIP is the first valid IP from PrimaryServiceClusterIPRange
```
Comment (dcbw) [inline cmd/kube-apiserver/app/options/options.go:72]: @pandaamanda the code doc should go right above the variable itself, can you move this down two lines?

Comment (pandaamanda) [reply cmd/kube-apiserver/app/options/options.go:72]: thanks. fix it.

**Pull request merged**